File: src/main/java/com/zeven/movie_api/vo/MovieVO.java
package com.zeven.movie_api.vo;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
@NoArgsConstructor
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class MovieVO {

    private String title;
    private String overview;
    @JsonProperty("adult")
    private Boolean isAdultContent;
}

File: APP/src/app/vos/MovieVO.ts
export class MovieVO{

  public title: string | null = null
  public overview: string | null = null
  public isAdultContent: string | null = null

  constructor(model: MovieVO) {
    this.initializer(model)
  }

  initializer(model: MovieVO | null){
    if(model != null){
      this.title = model.title
      this.overview = model.overview
      this.isAdultContent = model.isAdultContent
    }
    return this
  }
}
